common/opinion_modifiers/eaw_opinion_modifiers.txt

```diff
@@ -1,6 +1,6 @@
-
-# OPINION MODIFIERS
```

Comment:
I am just saying no to your removals here

Reply:
Only duplicate modifiers was removed, nothing more.